### DISCUSSION TITLE
Odd placement for the "delete vehicule" button

### DISCUSSION THREAD

**Discussion by redge76** [hargata/lubelog#637] [General]
My OCD is kicked by the odd placement of the "delete vehicule" button.  :wink:
It should be in the modal "edit vehicle" dialog opened with the yellow button.
Or the "edit vehicule" option should be in the "Manage vehicule" menu.

But both these functions in different places is an odd interface choice. There is only one option in the manage "vehicule menu".

(please consider this critic as constructive. I can live with the interface as it is :smiley:  . This project is so usefull for me)

**Comment by hargata** [comment]
Next Release:

![newdeletebutton](https://github.com/user-attachments/assets/05585f6d-da9e-4a3f-8701-d3ef0411bae4)

**Comment by hargata** [comment]
Released.
